Gordon360/Controllers/EmailsController.cs

```diff
@@ -15,9 +15,9 @@ public class EmailsController : GordonControllerBase
     {
         private readonly EmailService _emailService;
 
-        public EmailsController(CCTContext context)
+        public EmailsController(CCTContext context, IMembershipService membershipService)
         {
-            _emailService = new EmailService(context);
+            _emailService = new EmailService(context, membershipService);
```

Comment:
Yes, eventually all of our services should be injected as scoped dependencies. 

This is a change that we've been making incrementally, since it doesn't effect any other code - you can add it as a scoped dependency without changing the code where it's manually constructed at all. So in this case, since you're already changing how these services are constructed (and therefore have to change all the places where they're constructed) I think it makes sense to include it in this PR.

Reply:
Done.